[2023-08-22T04:55:47.934+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.third_task manual__2023-08-22T04:55:44.502745+00:00 [queued]>
[2023-08-22T04:55:47.963+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.third_task manual__2023-08-22T04:55:44.502745+00:00 [queued]>
[2023-08-22T04:55:47.965+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-08-22T04:55:48.011+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): third_task> on 2023-08-22 04:55:44.502745+00:00
[2023-08-22T04:55:48.022+0000] {standard_task_runner.py:57} INFO - Started process 8319 to run task
[2023-08-22T04:55:48.030+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'third_task', 'manual__2023-08-22T04:55:44.502745+00:00', '--job-id', '2273', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpuox1_1qb']
[2023-08-22T04:55:48.033+0000] {standard_task_runner.py:85} INFO - Job 2273: Subtask third_task
[2023-08-22T04:55:48.136+0000] {task_command.py:415} INFO - Running <TaskInstance: our_first_dag_v4.third_task manual__2023-08-22T04:55:44.502745+00:00 [running]> on host b0ce6883e4a1
[2023-08-22T04:55:48.324+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Techquier' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-22T04:55:44.502745+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-22T04:55:44.502745+00:00'
[2023-08-22T04:55:48.328+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-22T04:55:48.331+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task 3 and will be running after task 2']
[2023-08-22T04:55:48.354+0000] {subprocess.py:86} INFO - Output:
[2023-08-22T04:55:48.357+0000] {subprocess.py:93} INFO - hey, I am task 3 and will be running after task 2
[2023-08-22T04:55:48.359+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-22T04:55:48.440+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=third_task, execution_date=20230822T045544, start_date=20230822T045547, end_date=20230822T045548
[2023-08-22T04:55:48.527+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-22T04:55:48.576+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
